### PR TITLE
Apply DS to older fixes, improve color contrast

### PR DESCRIPTION
**Pull request by foxbunny**
- Apply DS features to some of the older fixes
- Remove all box-sizing overrides as it is now in the reset
- Remove DS' dependency on older SCSS modules
- Fix color contrast in the DS

Note: this is a maintenance patch. It does not address any specific issue nor does it add any significant features.

See #5967

### CODE REVIEW

**Comment by ThiefMaster** [comment]
I'm surprised we didn't have a sane `box-sizing` default before instead of setting it all over the place...

File: indico/modules/events/papers/templates/emails/judgment_to_author.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
     {% if paper.judgment_comment %}
         <dl>
             <dt style="width: 30%; float: left;">{% trans %}Comment:{% endtrans %}</dt>
-            <dd style="width: 70%; float: left; margin: 0; min-height: 20px; white-space: pre-wrap; word-break: break-all; padding-left: 20px; box-sizing: border-box;">
```
Comment (ThiefMaster) [inline indico/modules/events/papers/templates/emails/judgment_to_author.html:33]: These templates are for HTML emails (ie NO external CSS whatsoever) so I think removing box-sizing from any email template (`...../templates/emails/...`) may be risky - I doubt email clients default to border-box...

File: indico/web/client/styles/modules/_markdown.scss [hunk 2]
```diff
@@ -164,7 +164,6 @@ $display-preview-bg-color: white;
 }
 
 .wmd-button-bar {
-  box-sizing: content-box;
```
[the remark this answers is not in the document]
Comment (foxbunny) [reply indico/web/client/styles/modules/_markdown.scss:167]: > adjust the CSS

Probably better.

File: indico/modules/events/papers/templates/emails/judgment_to_author.html (hunk 1 above)
Comment (foxbunny) [reply indico/modules/events/papers/templates/emails/judgment_to_author.html:33]: Hm. Ok, let's revert those then.

**Pull request merged**